Towny/src/main/java/com/palmergames/bukkit/towny/object/Government.java

```diff
@@ -320,6 +321,10 @@ public String getMapColorHexCode() {
 		return mapColorHexCode;
 	}
 
+	public Color getColor() {
```

Comment:
getMapColor seems like a better name for this method, getColor is pretty abstract. Also a simple javadoc & nullability annotations would be nice. Color#decode can also throw a NumberFormatException, I think it would be best to catch it and just return null to signal that no hex code is set.